### PR TITLE
Create quick_sort.php

### PR DESCRIPTION
**Pull request by JulianMar**
Added quick_sort in php

### CODE REVIEW

**Comment by arnavb** [comment]
PHP code needs to be inside php tags, I think...

**Comment by JulianMar** [comment]
Yes you are correct. I added them to the file

File: code/sorting/quick_sort/quick_sort.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+functionfunction quick_sort( $array ) {
```
Comment (arnavb) [inline code/sorting/quick_sort/quick_sort.php]: Is this a syntax error? Why is `function` written twice?

Comment (JulianMar) [reply code/sorting/quick_sort/quick_sort.php]: I don't know how that happend but it is fixed

**Pull request closed without merging**